app/routes/languages.rb

```diff
@@ -10,6 +10,13 @@ class Languages < Core
         erb :"languages/index", locals: { active: active, inactive: inactive }
       end
 
+      get '/repository' do
+        tracks = X::Track.all
```

Comment:
@kytrinyx : We have done this way to maintain the consistency of the code. Same was followed in /languages as well.